data_ingestion.ipynb

```diff
@@ -22,7 +22,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    RASTER_NAMES = ['uc_davis_d18O_cel_ordinary_random_grouped_means', 'uc_davis_d18O_cel_ordinary_random_grouped_vars'] #@param
</pre><p>Is this list complete? If so, where does the rest of our data come from?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/197/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>